### PR TITLE
Update Release_Build_Creation.rst for Tested by Section

### PR DESCRIPTION
**Pull request by vaibhavparatwar**
Added information about verification done.

### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [ ] updated the docs
 - [ ] added a test

### CODE REVIEW

File: doc/Release_Build_Creation.rst [hunk 1]
```diff
@@ -105,3 +105,4 @@ Procedure
 Tested by:
 
 - Feb 12, 2021: Patrick Hession (patrick.hession@seagate.com) on a Windows laptop running VMWare Workstation Pro 16.
+- April 06, 2021: Vaibhav Paratwar (vaibhav.paratwar@seagate.com) on VM "LDRr1 - 2x CentOS 7.8 Shared Disks-20210329-232113"
```
Comment (mukul-seagate11) [inline doc/Release_Build_Creation.rst:108]: you should keep April entry at current i.e. above Feb entry

Comment (mukul-seagate11) [reply doc/Release_Build_Creation.rst:108]: and also fix DCO

Comment (vaibhavparatwar) [reply doc/Release_Build_Creation.rst:108]: ok. I will change

**Comment by vaibhavparatwar** [comment]
Will close this pull request and make separate one

**Pull request closed without merging**